zstacklib/zstacklib/utils/iscsi.py

```
@@ -34,23 +37,49 @@ def get_iscsi_device_serial(disk_path):
+    for dev in os.listdir(sys_block_path):
+        if not dev.startswith("dm-"):
+            continue
+        slaves_dir = os.path.join(sys_block_path, dev, "slaves")
+        if not os.path.isdir(slaves_dir):
+            continue
+        if disk_name in os.listdir(slaves_dir):
+            mapper_path = os.path.join("/dev/mapper", dev)
+            if os.path.exists(mapper_path):
+                return os.path.realpath(mapper_path)
+            else:
+                return os.path.join("/dev", dev)
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)